add_to_app/MultipleFluttersIos/MultipleFluttersIos.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
+	<true/>
+</dict>
+</plist>
```

Comment:
It didn't in my case.  Maybe because I made the project in a git repository and I had the checkmark off for adding a git repo.  Moved all of them and updated git ignores.

Reply:
Ah, could be the case. I vaguely remember this happening to me too.